src/test/jest.setup.js

```diff
@@ -0,0 +1,63 @@
+import 'whatwg-fetch';
+import { server } from './mocks/server';
+
+jest.mock('@unleash/proxy-client-react', () => ({
+  useUnleashContext: () => jest.fn(),
+  useFlag: jest.fn(() => true),
+}));
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+// Upgrading @patternfly/react-core caused propTypes error in Pf4FormTemplate
+// https://github.com/data-driven-forms/react-forms/issues/1352
+const filter1 = (args) => {
+  if (
+    args[2] ===
+    'Invalid prop `FormWrapper` supplied to `FormTemplate`, expected one of type [function].'
+  ) {
+    return [true, args[2]];
+  }
+  return [false, null];
+};
+
+class FilteredConsole {
+  constructor(console) {
+    this.console = console;
+    this.filters = [filter1];
+  }
+
+  logSuppressedError(err) {
+    this.console.info('Suppressed error: ', err);
+  }
+
+  filter(...args) {
+    for (let fn of this.filters) {
```

Comment:
It does and it's glorious! You'll need to switch `let` for `const` here to make the linter happy, but other than that looks good to me. Thank you!